.github/workflows/release.yml

```diff
@@ -63,26 +161,49 @@ jobs:
 
       - run: python -m pip install --upgrade pip wheel
       - run: python setup.py sdist
-
-      - run: python -m pip install dist/chromo-*
+      - run: python -m pip install --prefer-binary -v `echo dist/chromo-*`[test]
       - run: python -m pip install pytest
```

Comment:
The previous line 164 should also install pytest, so this line can be skipped, I think.

Reply:
Yes, right. It is removed.